WooCommerce/WooCommerceUITests/Tests/OrdersTests.swift

```diff
@@ -54,6 +56,17 @@ final class OrdersTests: XCTestCase {
             .verifySingleOrderScreenLoaded()
     }
 
+    func test_add_note_to_existing_order() throws {
+        let orders = try GetMocks.readOrdersData()
+
+        try TabNavComponent().goToOrdersScreen()
+            .tapOrder(byOrderNumber: orders[0].number)
```

Comment:
This is not PR-related, but something that caught my eye. Using `number` word in this context might give a false impression that this is about tapping by index (or it's just me 🤔). Using `id` property of mocks would be more descriptive in this sense, but again, this is out of scope.

Reply:
ah yes i could see how that could happen, since this is a property that is used across multiple tests, i'll update it in a follow-up PR (check if it's possible to use `id` instead of `number` and update for all tests)